sdk/python/pulumi_docker/docker.py

```diff
@@ -518,7 +516,7 @@ def login_to_registry(registry, log_resource: pulumi.Resource) -> Awaitable:
     else:
         log_ephemeral(f'Reusing existing login for {username}@{registry_name}', log_resource)
 
-    return login_result.login_command
```

Comment:
Does this work ? With `login_result = any(filter(...` above `login_result` is a boolean.